configs/sites/derecho/compilers.yaml

```diff
@@ -1,42 +1,47 @@
 compilers::
```

Comment:
Except for lines 10-12 adding the annoying deprecation warnings, the rest of the changes here are due to fixing the indentation only.

Reply:
So these will be set automatically whenever the `stack-intel` compiler gets loaded on Derecho - should be sufficient for UFS